package.json

```diff
@@ -5,7 +5,7 @@
   "bin": {
     "markdown-link-check": "markdown-link-check"
   },
-  "main": "index.js",
+  "main": "markdown-link-check",
```

Comment:
I think this is correct 🤷

Reply:
@tcort @smainil @BaseMax It would be good to get some other brains/eyes on this.

`index.js` (moved in this pull request to `src/index.js`) seems to contain things used in `src/markdown-link-check.js` and should not be used directly.

I think this change is correct, but maybe there is something I'm missing?